.php_cs

```diff
@@ -21,40 +21,80 @@ $config->setRules(
         'array_indentation' => true,
         'array_syntax' => ['syntax' => 'short'],
         'binary_operator_spaces' => true,
+        'blank_line_after_opening_tag' => true,
         'blank_line_before_statement' => [
-            'statements' => ['return', 'continue', 'case', 'default', 'declare', 'throw', 'try'],
+            'statements' => [
+                'break',
+                'case',
+                'continue',
+                'declare',
+                'default',
+                'die',
+                'do',
+                'exit',
+                'for',
+                'foreach',
+                'goto',
+                'if',
+                'include',
+                'include_once',
+                'require',
+                'require_once',
+                'return',
+                'switch',
+                'throw',
+                'try',
+                'while',
+                'yield',
+            ],
         ],
         'cast_spaces' => ['space' => 'none'],
-        'class_attributes_separation' => ['elements' => ['method']],
+        'class_attributes_separation' => ['elements' => ['method', 'property']],
+        'combine_consecutive_issets' => true,
+        'combine_consecutive_unsets' => true,
         'compact_nullable_typehint' => true,
         'concat_space' => ['spacing' => 'one'],
+        'date_time_immutable' => true,
```

Comment:
AFAIK, cs-fixer doesn't allow to ignore rules locally, so 👎 to this because what if I want to modify the date?

Reply:
https://derickrethans.nl/immutable-datetime.html

Reply:
Hmm. Not really a fan of mutable dates  - too many potential side effects.  Again opinionated, I'd suggest this is the saner option of the two?

Reply:
Let's have it then but I think it would be worth pointing out in the docs that some rules can change the code in a breaking manner and list them.